src/pages/billing/List.js

```diff
@@ -1,26 +1,58 @@
 import _ from 'lodash/fp'
+import * as qs from 'qs'
 import { Fragment } from 'react'
 import { a, div, h, span } from 'react-hyperscript-helpers'
-import { pure } from 'recompose'
-import { buttonPrimary, Clickable, PageBox, search, Select, spinnerOverlay } from 'src/components/common'
+import Interactive from 'react-interactive'
+import { buttonPrimary, buttonSecondary, Select, spinnerOverlay } from 'src/components/common'
 import { icon } from 'src/components/icons'
 import { validatedInput } from 'src/components/input'
 import Modal from 'src/components/Modal'
 import TopBar from 'src/components/TopBar'
 import { ajaxCaller } from 'src/libs/ajax'
 import * as Auth from 'src/libs/auth'
 import colors from 'src/libs/colors'
-import { reportError } from 'src/libs/error'
+import { withErrorReporting } from 'src/libs/error'
 import { formHint, RequiredFormLabel } from 'src/libs/forms'
 import * as Nav from 'src/libs/nav'
 import * as StateHistory from 'src/libs/state-history'
 import * as Style from 'src/libs/style'
 import * as Utils from 'src/libs/utils'
 import { Component } from 'src/libs/wrapped-components'
-import NewAccountModal from 'src/pages/billing/NewAccountModal'
+import ProjectDetail from 'src/pages/billing/Project'
 import validate from 'validate.js'
 
 
+const styles = {
+  tab: {
+    display: 'flex', alignItems: 'center', fontSize: 16, height: 50, padding: '0 2rem',
+    fontWeight: 500, overflow: 'hidden', borderBottom: `1px solid ${colors.grayBlue[2]}`,
+    borderRightStyle: 'solid'
+  }
+}
+
+const ProjectTab = ({ project: { projectName, role, creationStatus }, isActive }) => {
+  const projectReady = creationStatus === 'Ready'
+  const statusIcon = icon(creationStatus === 'Creating' ? 'loadingSpinner' : 'error-standard',
+    { style: { color: colors.green[0], marginRight: '1rem', marginLeft: '0.5rem' } })
+
+  return !!_.includes('Owner', role) && projectReady ? h(Interactive, {
```

Comment:
Nit: I think this would be more readable with `_.includes('Owner', role) && projectReady` extracted into a local `const`. Otherwise, it initially looks like the `boolExpression && thingToConditionallyRender()` pattern... until you see the `?` and then have to think about operator precedence between `&&` and `? :`.

Reply:
Funny you say that, I had `_.includes('Owner', role) && projectReady` in a local `const` originally but @zarsky-broad recommended inlining the condition because it was only used once. I'm going to keep it as is this time in order to expedite finishing this PR, but that's good feedback.